### PR TITLE
Add "metanodes" filter to nodes endpoint

### PR DESCRIPTION
**Pull request by dongbohu**

### CODE REVIEW

File: dj_hetmech_app/serializers.py [hunk 1]
```diff
@@ -13,18 +13,8 @@ class Meta:
...
-    It is defined specifically for PathCountDgpSerializer.
-    """
-    class Meta:
-        model = Metapath
-        fields = ('abbreviation', 'name', )
-
-
 class PathCountDgpSerializer(serializers.ModelSerializer):
     dgp = DgpSerializer()
-    metapath = CompactMetaPathSerilizer()
```
Comment (dhimmel) [inline dj_hetmech_app/serializers.py:27]: You probably also have to update the polisher to have the key rather than the dict as the metapath field?

**Review by dhimmel — COMMENTED** [review]
Metanode filter looks good to me.

Will this implementation result in improved computational efficiency for searching nodes when a metanode filter is applied?

File: dj_hetmech_app/serializers.py (hunk 1 above)
Comment (dongbohu) [reply dj_hetmech_app/serializers.py:27]: Already did.

**Comment by dongbohu** [comment]
@dhimmel About the performance after the filter is added, yes, it should be faster because the queryset will be smaller when the filter is applied.

**Review by dhimmel — APPROVED** [review]

**Pull request merged**